frame/evm-system/src/tests.rs

```diff
@@ -100,9 +106,9 @@ fn remove_account_fails() {
 		let account_id = H160::from_str("1000000000000000000000000000000000000001").unwrap();
 
 		// Invoke the function under test.
-		assert_noop!(
+		assert_eq!(
```

Comment:
Why switch from `assert_noop` to `assert_eq`?

Reply:
DispatchError was used before, now `AccountRemovalStatus` and `AccountCreationStatus` have been introduced and used for `create_account`, `remove_account` respectively.

Reply:
We should wrap the assertion with `assert_state_noop!` then

Reply:
https://github.com/humanode-network/frontier/pull/111/commits/67227cb742e4268d8758c5ea1d9bc20f44ad4958